src/GDS/Mapper/RESTv1.php

```diff
@@ -401,6 +401,11 @@ protected function createPropertyValue(array $arr_field_def, $mix_value)
         }
         $obj_property_value->excludeFromIndexes = !$bol_index;
 
+        if (is_null($mix_value)) {
```

Comment:
Prefer `null === $mix_value` for performance (micro optimisation).

Reply:
Done